styles/lists.css

```diff
@@ -22,3 +22,27 @@
 .listLink {
   text-decoration: none;
 }
+
+.itemMassActions {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: var(--border-color);
+  padding: 0.5rem;
+  border-radius: 8px;
+  margin-top: 1rem;
+}
+
+.selectAllInputContainer label {
+  display: flex;
+  align-items: center; /* Vertically center the checkbox and text */
+  font-size: 18px;
+}
+
+.selectAllInputContainer input[type='checkbox'] {
+  width: 20px; /* Adjust width */
```

Comment:
Setting up the position of the checkbox was really finnicky here - but remove the notes to self